rpc/handlers.go

```diff
@@ -65,12 +65,15 @@ var (
 	ErrUnsupportedTxVersion            = &jsonrpc.Error{Code: 61, Message: "the transaction version is not supported"}
 	ErrUnsupportedContractClassVersion = &jsonrpc.Error{Code: 62, Message: "the contract class version is not supported"}
 	ErrUnexpectedError                 = &jsonrpc.Error{Code: 63, Message: "An unexpected error occurred"}
+	ErrTooManyBlocksBack               = &jsonrpc.Error{Code: 68, Message: "Cannot go back more than 1024 blocks"}
```

Comment:
do you think it's worth to pass actual constant here i.e.:
```
Message: fmt.Sprintf("Cannot go back more than %d blocks", maxBlocksBack)
```

Reply:
Done